tools/doc/html.js

```diff
@@ -8,6 +8,15 @@ var typeParser = require('./type-parser.js');
 
 module.exports = toHTML;
 
+// customized heading without id attribute
+var renderer = new marked.Renderer();
```

Comment:
Yes it's a problem. I think I'd better change the template string to string replace in order to keep the style consistent.